app/models/bot_user.rb

```diff
@@ -315,7 +315,11 @@ def self.notify_bots(event, team_id, object_class, object_id, target_bot, reques
     return if object.nil? || team.nil?
     team.team_bot_installations.each do |team_bot_installation|
       bot = team_bot_installation.bot_user
-      bot.notify_about_event(event, object, team, team_bot_installation) if bot.subscribed_to?(event) && (target_bot.blank? || bot.id == target_bot.id)
+      begin
+        bot.notify_about_event(event, object, team, team_bot_installation) if bot.subscribed_to?(event) && (target_bot.blank? || bot.id == target_bot.id)
+      rescue StandardError => e
```

Comment:
**question** what is this rescue intending to catch here? I'm curious because right now we seem to be catching all of the errors arising from `run` in the auto-tagger bot above and not re-raising, so shouldn't be introducing any new errors at this level, I think

Reply:
This change is unrelated to Tagger. Tagger does catch all errors as you note, but other Bots do not. In particular, Alegre does not and an exception in Alegre was causing Tagger not to run in some instances. The goal of the change here is to ensure we run all bots that should run regardless of whether they all succeed.